src/components/enterprise-page/EnterprisePage.test.jsx

```diff
@@ -11,47 +11,51 @@ import * as hooks from './data/hooks';
 const mockUser = {
   profileImage: 'http://fake.image',
 };
-jest.mock('@edx/frontend-platform/auth');
+jest.mock('@edx/frontend-platform/auth', () => ({
+  ...jest.requireActual('@edx/frontend-platform/auth'),
+  getAuthenticatedUser: () => mockUser,
+}));
 jest.mock('react-router-dom', () => ({
   useParams: jest.fn().mockReturnValue({ enterpriseSlug: 'test-enterprise-slug' }),
 }));
 
 describe('<EnterprisePage />', () => {
   beforeEach(() => {
     jest.clearAllMocks();
+    initializeMockApp();
   });
   describe('renders loading state', () => {
     it('while fetching enterprise config', () => {
-      jest.spyOn(auth, 'getAuthenticatedUser').mockImplementation(() => mockUser);
+      // jest.spyOn(auth, 'getAuthenticatedUser').mockImplementation(() => mockUser);
       // mock hook as if async call to fetch enterprise config is still resolving
       jest.spyOn(hooks, 'useEnterpriseCustomerConfig').mockImplementation(() => [undefined, undefined]);
-      const wrapper = mount(<EnterprisePage><div className="did-i-render" /></EnterprisePage>);
+      const wrapper = mount(<IntlProvider locale="en"><EnterprisePage><div className="did-i-render" /></EnterprisePage></IntlProvider>);
```

Comment:
nit: completely fine to defer, but curious if you think it'd be worth adding a `EnterprisePageWrapper` component that includes both `IntlProvider` and `EnterprisePage` as a single component to DRY this up a bit?